### ISSUE TITLE
Imaging GUI: Sort the processes in Run tab's Reconstruction Jobs

### ISSUE THREAD

**Issue by DimitarTasev** [mantidproject/mantid#18143]
```
Also the ability to sort reconstruction jobs esp by ID and Status
```

Check what storage type is used to create the table and if Qt has any methods allowing it.

**Comment by DimitarTasev** [comment]
Covered by a fix in https://github.com/mantidproject/mantid/pull/18095

**Comment by DimitarTasev** [comment]
Re-opened as setting sorting to enabled in QtCreator causes other problems:
- Sorting is inconsistent and clunky, it does not work properly, and will often show the items in a wrong order
- Mantid will crash with `ASSERT failure in QPersistentModelIndex::~QPersistentModelIndex: "persistent model indexes corrupted", file kernel\qabstractitemmodel
.cpp, line 544`

**Comment by DimitarTasev** [comment]
Closed via #19124 *All other issues concerning the Tomography GUI are now invalidated.*
